src/components/WeatherCard/WeatherCard.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import * as S from './style';
+
+/* Import Components */
+import TransparentBox from '../TransparentBox';
```

Comment:
just import everthing from './components

Reply:
I need create new file (`index.jsx`) in components folder and export all components from there ?